__tests__/app.test.js

```diff
@@ -181,3 +183,75 @@ describe("GET /api/articles", () => {
       });
   });
 });
+
+describe("GET /api/articles/:articleid/comments", () => {
+  test("should return 200", () => {
+    return request(app).get("/api/articles/1/comments").expect(200);
+  });
+  test("should return an array of object that all the comments of the given article ", () => {
+    const articleId = 1
+    const count = commentData.reduce((counter, obj) => {
+      if (obj.article_id === articleId) counter += 1
+      return counter;
+    }, 0)
```

Comment:
This works here, but as we have very simple test data, it's okay to just cound the comments for an article and use that number instead of writing all this code to get that number. Less code -> less chance for bugs.